templates/saphana.tpl.yaml

```diff
@@ -0,0 +1,188 @@
+          annotations:
+            vm.kubevirt.io/os: "{{ os }}"
+            vm.kubevirt.io/workload: "{{ item.workload }}"
+            vm.kubevirt.io/flavor: "{{ item.flavor }}"
+          labels:
+            kubevirt.io/domain: ${NAME}
+            kubevirt.io/size: {{ item.flavor }}
+        spec:
+          hostname: ${NAME}
+          domain:
+            ioThreadsPolicy: auto
+            cpu:
+              cores: {{ '${{CPU_CORES}}' }}
+              threads: {{ '${{CPU_THREADS}}' }}
+              sockets: {{ '${{CPU_SOCKETS}}' }}
+              model: host-passthrough
+              isolateEmulatorThread: true
+              dedicatedCpuPlacement: true
+              features:
+                - name: "invtsc"
+                  policy: "require"
+              numa:
+                guestMappingPassthrough : {}
+            devices:
+              blockMultiQueue: true
+              networkInterfaceMultiqueue: true
+              disks:
+                - disk:
+                    bus: virtio
+                  dedicatedIOThread: true
+                  name: ${NAME}
+                - disk:
+                    bus: virtio
+                  name: cloudinitdisk
+                - disk:
+                    bus: virtio
+                  name: downwardmetrics
+              interfaces:
+                - masquerade: {}
+                  name: default
+                - name: sriov-net1
+                  sriov: {}
+                - name: sriov-net2
+                  sriov: {}
+                - name: sriov-net3
+                  sriov: {}
+            machine:
+              type: ""
+            resources:
+              requests:
+                memory: ${MEMORY_OVERHEAD}
+            memory:
+              guest: ${MEMORY}
+              hugepages:
+                pageSize: ${HUGEPAGES_PAGE_SIZE}
+          terminationGracePeriodSeconds: 0
+          volumes:
+            - dataVolume:
+                name: ${NAME}
+              name: ${NAME}
+            - cloudInitNoCloud:
+                userData: |-
+                  #cloud-config
+                  user: {{ cloudusername }}
+                  password: ${CLOUD_USER_PASSWORD}
+                  chpasswd: { expire: False }
+              name: cloudinitdisk
+            - emptyDisk:
+                capacity: 2Gi
+              name: emptydisk
+            - downwardMetrics: {}
+              name: downwardmetrics
+          networks:
+            - name: default
+              pod: {}
+            - multus:
+                networkName: ${SRIOV_NETWORK_NAME1}
+              name: sriov-net1
+            - multus:
+                networkName: ${SRIOV_NETWORK_NAME2}
+              name: sriov-net2
+            - multus:
+                networkName: ${SRIOV_NETWORK_NAME3}
+              name: sriov-net3
+          nodeSelector:
+            kubernetes.io/hostname: ${TARGET_NODE_NAME}
+parameters:
+  - description: Name for the new VM
+    name: NAME
```

Comment:
Do we want to have some default name?

Reply:
I think specifying a name is a requierment. But now that you bring it up, probably not specifying a default for the target node name may make sense. What do you think?

Reply:
Also removed the default value from TARGET_NODE_NAME.